packages/java/tests/pom.xml

```diff
@@ -281,24 +281,27 @@
             </modules>
         </profile>
 
+        <!-- the spring module constains submodules to run with different profiles -->
```

Comment:
Shouldn't that be handled by the `it-modules` profile above?

Reply:
the above one will be deactived during PR validation. check here 
https://github.com/vaadin/hilla/blob/main/.github/workflows/validation.yml#L246